src/components/templates/ProductTemplate/ProductTemplate/ProductTemplate.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import ProductTemplate from '..';
+
+describe('<ProductTemplate />', () => {
+  it('renders with title and children but without subtitle', () => {
+    const { container } = render(<ProductTemplate title="ProductTemplate title">All the children</ProductTemplate>);
+    expect(container).toMatchSnapshot();
+  });
+
+  it('renders with title, children and subtitle', () => {
+    const { container } = render(<ProductTemplate title="ProductTemplate title">All the children</ProductTemplate>);
+    expect(container).toMatchSnapshot();
```

Comment:
I don't need we need that amount of snapshots. I would suggest to test for the presence of the sub components and leave only one snapshot per component.